e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs

```diff
@@ -137,5 +143,20 @@ public void Dispose()
 
         string IOpcPlcConfig.Urls => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PLC_SIMULATION_URLS,
             () => throw new Exception("Semicolon separated list of URLs of OPC-PLCs is not provided."));
+
+        string IContainerRegistryConfig.DockerServer => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_DOCKER_SERVER,
+            () => throw new Exception("Server name is not provided"));
+
+        string IContainerRegistryConfig.DockerUser => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_DOCKER_USER,
+            () => throw new Exception("Username is not provided"));
+
+        string IContainerRegistryConfig.DockerPassword => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_DOCKER_PASSWORD,
+            () => throw new Exception("Password is not provided"));
+
+        string IContainerRegistryConfig.ImagesNamespace => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_IMAGES_NAMESPACE,
+            () => throw new Exception("Namespace is not provided"));
+
+        string IContainerRegistryConfig.ImagesTag => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_IMAGES_TAG,
+            () => Assembly.GetExecutingAssembly().GetReleaseVersion().ToString(3));
```

Comment:
1. None of those environment variables are set anywhere, so all of member initializers will fall back to `throw` case. And as a result none of the test will be able to start.

2. All of those values should be aligned with what IAI deploys. Otherwise we will end up deploying `2.7.170` version of the platform and testing `2.8.x` version of OPC Publisher against it, which is incorrect. So we will have to define which version of microservices IAI should deploy through configuration and then propagate those values down to E2E test stage.

Reply:
fixed